app/views/solution-card.test.js

```diff
@@ -112,7 +133,30 @@ describe('solution-card', () => {
         done();
       });
   });
+  {/* <div data-test-id="solution-card-name" class="nhsuk-u-margin-top-5">
+<a href="/view-solution/{{ solution.id }}">View the solution</a>
+</div> */}
+  it('should render the View the solution link', (done) => {
+    const context = {
+      solution: {
+        organisationName: 'some organisation name',
+      },
+    };
+
+    const app = createDummyApp(context);
+    request(app)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
 
+        const solutionOrganisationName = $('[data-test-id="solution-card-organisation"]');
+
+        expect(solutionOrganisationName.length).toEqual(1);
+        expect(solutionOrganisationName.text().trim()).toEqual('some organisation name');
+
+        done();
+      });
+  });
```

Comment:
Thought I'd already put an organisation name test in? Also the test description does not match the implementation.

Reply:
let me check ...

Reply:
yeah this was a duplicate :) deleted